packages/core/core/src/registries/config.ts

```diff
@@ -8,7 +9,7 @@ export default (initialConfig = {}, strapi?: Strapi | LoadedStrapi): ConfigProvi
 
   // Accessing model configs with dot (.) was deprecated between v4->v5, but to avoid a major breaking change
   // we will still support certain namespaces, currently only 'plugin.'
-  const transformDeprecatedPaths = (path: PropertyName) => {
+  const transformPathString = (path: string) => {
     if (isString(path) && path.startsWith('plugin.')) {
```

Comment:
isString is not necessary anymore :D

Reply:
what if we want to be REALLLY sure?

Reply:
it's an internal function that you call within Ts context. In case it was a public utility sure